core/controllers/base.py

```diff
@@ -156,7 +157,8 @@ def __init__(self, request, response):  # pylint: disable=super-init-not-called
             user_settings = user_services.get_user_settings(
                 self.user_id, strict=False)
             if user_settings is None:
-                email = current_user_services.get_user_email(self.user)
+                email = current_user_services.get_user_email(
```

Comment:
In line 156,
what should be `current_user_services.get_user_email` be when `current_user_services.get_current _user` is `None`.
Currently the function only returns user.email()